KenticoCloud.Delivery/DeliveryClient.cs

```diff
@@ -528,16 +560,27 @@ public DeliveryClient(string projectId, string previewApiKey) : this(new Deliver
                 message.Headers.Authorization = new AuthenticationHeaderValue("Bearer", _deliveryOptions.PreviewApiKey);
             }
 
-            var response = await HttpClient.SendAsync(message);
+            return HttpClient.SendAsync(message);
+        }
 
-            if (response.StatusCode == HttpStatusCode.OK)
+        private async Task<JObject> GetResponseContent(HttpResponseMessage httpResponseMessage)
+        {
+            if (httpResponseMessage?.StatusCode == HttpStatusCode.OK)
             {
-                var content = await response.Content.ReadAsStringAsync();
+                var content = await httpResponseMessage.Content?.ReadAsStringAsync();
 
                 return JObject.Parse(content);
             }
 
-            throw new DeliveryException(response, await response.Content.ReadAsStringAsync());
+            string faultContent = null;
+
+            // The null-coallescing operator cannot be used, hence "if" statement.
```

Comment:
why is that? it seems to work for me....
either explain the reasons clearly in the comment or drop the comment

Reply:
oh, the tests are failing...but still, please explain the reasons for the future generations

Reply:
Done